assets/javascript/autocomplete.js

```diff
@@ -183,5 +183,4 @@ $(function () {
 
 
 
-}); //end on ready function
-
+}); //end on ready function
```

Comment:
On GitHub I see a warning because there isn't a newline at the end of line.

Reply:
Thanks, I'm trying to make sure those are removed from our project, but it doesn't really matter.